keps/sig-storage/3107-csi-nodeexpandsecret/README.md

```diff
@@ -220,53 +237,137 @@ N/A
 
 ### Rollout, Upgrade and Rollback Planning
 
-TBD
-
 ###### How can a rollout or rollback fail? Can it impact already running workloads?
 
-TBD
+<!--
+Try to be as paranoid as possible - e.g., what if some components will restart
+mid-rollout?
 
-###### What specific metrics should inform a rollback?
+Be sure to consider highly-available clusters, where, for example,
+feature flags will be enabled on some API servers and not others during the
+rollout. Similarly, consider large clusters and how enablement/disablement
+will rollout across nodes.
+-->
 
-TBD
+###### What specific metrics should inform a rollback?
 
+<!--
+What signals should users be paying attention to when the feature is young
+that might indicate a serious problem?
+-->
```

Comment:
Are there any system-wide metrics? Do CSI drivers need to be publishing their own metrics?

Reply:
Having a metric for total number of calls that require a secretRef + a metric for where the secret ref was bad (error) might be sufficient?

Reply:
CSI drivers do have metrics. however this particular feature does not need any special metrics being completely optional to an RPC call and also by following other secrets in CSI spec ( provision/snapshot/controller expand..etc) which dont have any metrics either